recipes/folly/all/conanfile.py

```diff
@@ -83,6 +83,13 @@ def _strip_options_requirements(self):
     def _required_boost_components(self):
         return ["context", "filesystem", "program_options", "regex", "system", "thread"]
 
+    # TODO: Remove CMake from build_requirements when folly/2019.10.21.00 is removed from CCI
+    # Bump max. CMake version to 3.16.2 to fix the Linux build
```

Comment:
This is not a bump this is a downgrade. Why did you not go for CMake `3.19.2`?

Reply:
Because this CMake version already worked on CCI for folly and because the CMake version must be < 3.18 to compile the old folly.

Reply:
If this is the case, a CMake < 3.18 version needs to be permanent added. Either way, this comment is misleading.

Reply:
Ah, you updated it already, nvm :)

Reply:
Maybe I should click on Resolve conversation next time. :-)